src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,75 +1,167 @@
+          / Metrics.Chassis.TICKS_PER_REVOLUTION;
+    }
+  }
+
+  public static class DriveConstants {
+    public static final boolean kGyroReversed = false;
+    public static final double ksVolts = -1;
+    public static final double kvVoltSecondsPerMeter = -1;
+    public static final double kaVoltSecondsSquaredPerMeter = -1;
+    public static final double kTrackwidthMeters = -1;
+    public static final DifferentialDriveKinematics kDriveKinematics = new DifferentialDriveKinematics(
+        kTrackwidthMeters);
+    public static final double kPDriveVel = -1;
+  }
+
+  public static class AutoConstants {
+    public static final double kMaxSpeedMetersPerSecond = -1;
+    public static final double kMaxAccelerationMetersPerSecondSquared = -1;
+    public static final double kRamseteB = -1;
+    public static final double kRamseteZeta = -1;
+  }
+
+  public static class PID {
+    public static class Flywheel {
+      public static final double P = 0;
+      public static final double I = 0;
+      public static final double D = 0;
+    }
+    public static class Drive {
     }
 
-    public static class Metrics {
-        public static class Chassis {
-            public static final double TICKS_PER_REVOLUTION = -1; // TODO: CHANGE CONSTS
-            public static final double DIAMETER_INCHES = -1;
-            public static final double CIRCUMFERENCE_INCHES = Metrics.Chassis.DIAMETER_INCHES * Math.PI;
-            public static final double TICKS_PER_INCH = Metrics.Chassis.TICKS_PER_REVOLUTION
-                    / Metrics.Chassis.CIRCUMFERENCE_INCHES;
-            public static final double DISTANCE_FRONT_BACK = -1;
-            public static final double DISTANCE_SIDE_SIDE = -1;
-            public static final double INCHES_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_INCHES
-                    / Metrics.Chassis.TICKS_PER_REVOLUTION;
-        }
+    public static class Turn {
     }
 
-    public static class PID {
-        public static class Drive {
-        }
+  }
 
-        public static class Turn {
-        }
 
-    }
+  public static class Component {
+    public static Intake intake;
+    public static Indexer indexer;
+    public static Flywheel flywheel;
+    public static Shooter shooter;
 
-    public static class Component {
-    }
+    public static SolenoidSubsystem intakeSolenoid;
+    public static SolenoidSubsystem flipperSolenoid;
+    public static SolenoidSubsystem shooterAimSolenoid;
 
-    public static class Input {
-    }
+    public static Motor intakeRollerMotor;
+    public static Motor funnelMotor;
+    public static Motor liftBeltMotor;
+    public static Motor runUpBeltMotor;
+    public static Motor flywheelMotorA;
+    public static Motor flywheelMotorB;
 
-    public static class HumanInput {
-        public static class Driver {
-            public static CustomXbox xbox;
-        }
+    public static CustomCANCoder flywheelEncoder;
```

Comment:
should be the integrated one on the falcon 500 (in `talon-encoder-reworks` in `standard`)